docs/source/projections/eqc.rst

```diff
@@ -8,3 +8,31 @@ Equidistant Cylindrical (Plate Caree)
    :scale: 50%
    :alt:   Equidistant Cylindrical (Plate Caree)  
 
+Marinus of Tyre was attributed this equirectangular projection :
+
+.. math::
+
+   x = (\lambda - \lambda0) cos \phi0 
```

Comment:
Sorry, I need to remove \lambda0, as proj4 do not implement it.

I can change \phi0 to \phi1 if it sounds better, or something ?